ui/src/main/java/edu/wpi/grip/ui/GripPreloader.java

```diff
@@ -0,0 +1,32 @@
+package edu.wpi.grip.ui;
+
+import javafx.application.Preloader;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Scene;
+import javafx.stage.Stage;
+import javafx.stage.StageStyle;
+
+public class GripPreloader extends Preloader {
+
+  private Stage preloaderStage;
+
+  @Override
+  public void start(Stage preloaderStage) throws Exception {
```

Comment:
You don't have to throw the `Exception` even if the parent class does. I'd rather see explicitly what methods this method actually throws.
Try removing it and adding only the exceptions that get thrown and see if it still compiles. I bet it will.